app/services/duplicate_upload_verification_service.rb

```diff
@@ -16,14 +24,13 @@ def duplicates
     duplicate_file_sets.map(&:parent)
   end
 
-  # @return [Array<FileSet>]
-  def duplicate_file_sets
-    return [] if file.nil?
-    FileSet.where(digest_ssim: fedora_shasum)
-  end
-
   private
 
+    # returns array of one parent asset
+    def parent_asset
```

Comment:
@RudyOnRails I don't see this method getting called anywhere.

Reply:
Good catch.